spec/order_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "order"
+
```

Comment:
As you are using 'order = Order.new' in each test, you could perhaps declare it at the beginning of the test as a 'let' statement.